src/back/controllers/api/routes/challenges/post.js

```diff
@@ -1,17 +1,42 @@
 'use strict';
 
 const Challenge = require('../../models/challenge');
+const request = require('request');
 
-const post = (req, res) => {
-    const model = new Challenge(req.body);
+const newPost = (req, res) => {
+    const baseHostingUrl = 'http://ec2-52-35-85-119.us-west-2.compute.amazonaws.com:8080/';
+    const url = `${baseHostingUrl}picture`;
+    const body = req.body;
+    const imageBuffer = new Buffer(body.image, 'base64');
```

Comment:
I was thinking about that, but unfortunately fetch have no ability to work with streams and I don't know whether external hosting supports streams.